src/zc/buildout/pip_support.py

```diff
@@ -0,0 +1,8 @@
+try: 
+    # support pip>=19.0
+    from pip._internal.cli.main import main
+    pip_install_cmd = 'from pip._internal.cli.main import main; main()'
+except ImportError:
+    # support pip>=10.0
+    from pip._internal import main
+    pip_install_cmd = 'from pip._internal import main; main()'
```

Comment:
This file needs a comment like `# The pip developers don't want us to import these internal modules, we know, we chose to disregard their advice because ...` and then explain why we're doing what we're not supposed to be doing.